docs/content.zh/release-notes/flink-1.19.md

```diff
@@ -0,0 +1,354 @@
+We strongly recommend that users and developers use the ConfigOption-based get and set methods directly.
+
+#### Support System out and err to be redirected to LOG or discarded
+
+##### [FLINK-33625](https://issues.apache.org/jira/browse/FLINK-33625)
+
+`System.out` and `System.err` output the content to the `taskmanager.out` and `taskmanager.err` files.
+In a production environment, if flink users use them to print a lot of content, the limits of yarn
+or kubernetes may be exceeded, eventually causing the TaskManager to be killed. Flink supports
+redirecting the `System.out` and `System.err` to the log file, and the log file can be rolled to
+avoid unlimited disk usage.
+
+#### Support standard YAML for FLINK configuration
+
+##### [FLINK-33297](https://issues.apache.org/jira/browse/FLINK-33297)
+
+Starting with Flink 1.19, Flink has officially introduced full support for the standard YAML 1.2
+syntax ([FLIP-366](https://cwiki.apache.org/confluence/display/FLINK/FLIP-366%3A+Support+standard+YAML+for+FLINK+configuration?src=contextnavpagetreemode)). The default configuration file has been changed to `config.yaml` and placed in the
+`conf/` directory. Users should directly modify this file to configure Flink.
+If users want to use the legacy configuration file `flink-conf.yaml`, they need to copy this file
+into the `conf/` directory. Once the legacy configuration file `flink-conf.yaml` is detected, Flink
+will prioritize using it as the configuration file. In the upcoming Flink 2.0, the `flink-conf.yaml`
+configuration file will no longer work.
+More details could be found at [flink-configuration-file](https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/config/#flink-configuration-file).
+
+#### Add config options for administrator JVM options
+
+##### [FLINK-33221](https://issues.apache.org/jira/browse/FLINK-33221)
+
+A set of administrator JVM options are available, which prepend the user-set extra JVM options for
+platform-wide JVM tuning.
+
+#### Flink Job stuck in suspend state after losing leadership in HA Mode
+
+##### [FLINK-34007](https://issues.apache.org/jira/browse/FLINK-34007)
+
+Fixed a bug where the leader election wasn't able to pick up leadership again after renewing the
+lease token caused a leadership loss. This required `fabric8io:kubernetes-client` to be upgraded
+from v6.6.2 to v6.9.0.
+
+#### Support dynamic source parallelism inference for batch jobs
+
+##### [FLINK-33768](https://issues.apache.org/jira/browse/FLINK-33768)
+
+In Flink 1.19, we have supported dynamic source parallelism inference for batch jobs, which allows
+source connectors to dynamically infer the parallelism based on the actual amount of data to consume.
+This feature is a significant improvement over previous versions, which only assigned a fixed default
+parallelism to source vertices.
+Source connectors need to implement the inference interface to enable dynamic parallelism inference.
```

Comment:
`Source connectors need to implement the interface ...`. There is a useless word 'inference'.

Reply:
@ruanhang1993 IIUC, the 'inference interface' refers to  `DynamicParallelismInference`(https://cwiki.apache.org/confluence/display/FLINK/FLIP-379%3A+Dynamic+source+parallelism+inference+for+batch+jobs), so it's needed here.